extension/src/experiments/columns/quickPick.ts

```diff
@@ -14,11 +15,13 @@ export const pickFromColumnLikes = (
     return Toast.showError('There are no columns to select from.')
   }
   return quickPickValue<ColumnLike>(
-    columnLikes.map(columnLike => ({
-      description: columnLike.path,
-      label: columnLike.label,
-      value: columnLike
-    })),
+    columnLikes
+      .filter(columnLike => columnLike.path !== timestampColumn.path)
```

Comment:
[C] I do not think we want to filter this here. We will actually want to make it possible to sort/filter by timestamp in the future.

Reply:
> [C] I do not think we want to filter this here. We will actually want to make it possible to sort/filter by timestamp in the future.

I did test out sorting with timestamp but it wasn't working so I disabled both for now :)